src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -236,8 +415,88 @@ describe('sidebar/store/modules/annotations', function() {
       );
     });
 
+    it('should set basic default properties on a new/empty annotation', () => {
+      store.dispatch(actions.createAnnotation({ id: 'myID' }, now));
```

Comment:
Using `store.dispatch(...)` does match the convention of some of the other tests in this file, but in most recently-written store code we have been instead using the convenience methods added by the `createStoreFromModules` helper. See for example `real-time-test.js`. I would suggest following that pattern as it is more succinct and moves the code overall towards consistency.

Reply:
Sorry, I'm confused. There is a store module called `real-time-updates`—if I look in tests for that module, I see:

```
    store = createStore(
      [realTimeUpdates, annotations, selection],
      [fakeSettings]
    );
```

whereas `annotations-test` uses:

```
   createStoreFromModules(
    [annotations, selection, drafts, groups, session, viewer],
    [{}]
  );
```

Which one of these patterns is approved? I can't quite parse your comment.

Reply:
I think its confusing because the same function is imported with a different name in a few places. `createStoreFromModules` inside annotations-test is the same function as `createStore` inside real-time-updates. That is confusing...

createStoreFromModule is just a pseudo name for it that added some extra indirection that I'm not sure helps anything. I think that was done because the store was re-created a few times in that file and its wrapped so we don't have to pass all those module params each time.

I would suggestion doing this
`const createStore = require('../../create-store');
`

```

/**
 * Create a Redux store which handles annotation, selection and draft actions.
 */
function createTestStore() {
  return createStore(
    [annotations, selection, drafts, groups, session, viewer],
    [{}]
  );
}

```

Then use createTestStore() everywhere else in the file.


I think what Rob was implying with the dispatch thing was doing this

```store.createAnnotation({ id: 'myID' }, now);```
instead of this
```store.dispatch(actions.createAnnotation({ id: 'myID' }, now));```

Both are identical, the former just wraps the dispatch call and is a little more convenient to use.